### PR TITLE
[MRG] Fixing documentation for plot_seeg and adding some tutorial links.

### PR DESCRIPTION
**Pull request by adam2392**
#### Reference issue
Closes: #8428 

#### What does this implement/fix?
Links the `plot_seeg.py` tutorial to other tutorials that will put into context how to:

1. work with BEM surfaces
2. work with different volumetric plotting options
3. work with FreeSurfer transformations and their different volumes

### CODE REVIEW

**Comment by drammock** [comment]
@larsoner the CircleCI error is one I've not seen before, and doesn't seem likely to be directly related to the changes in this PR:

```
generating gallery for auto_tutorials/misc... [100%] plot_seeg.py
  11.1 s : 423.2 MB   
Exception ignored in: <finalize object at 0x7f1b9406b960; dead>
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/weakref.py", line 566, in __call__
    return info.func(*info.args, **(info.kwargs or {}))
  File "/usr/local/lib/python3.8/tempfile.py", line 819, in _cleanup
    _warnings.warn(warn_message, ResourceWarning)
ResourceWarning: Implicitly cleaning up <TemporaryDirectory '/tmp/tmpryze7ed4'>
```

...looks related to your recent work with @GuillaumeFavelier on CircleCI memory management. Do either of you recognize the problem from the traceback?

**Comment by larsoner** [comment]
> the CircleCI error is one I've not seen before, an

That particular warning does not show up as an error. I think it's related to `matplotlib` but I don't see it locally so I expect it to automagically disappear once 3.4 is out. The real errors are these:
```
/home/circleci/project/doc/auto_tutorials/misc/plot_seeg.rst:793: WARNING: py:obj reference target not found: stc.data[idx]
/home/circleci/project/doc/auto_tutorials/misc/plot_seeg.rst:793: WARNING: py:obj reference target not found: idx
/home/circleci/project/doc/auto_tutorials/misc/plot_seeg.rst:61117: WARNING: py:obj reference target not found: fsaverage
/home/circleci/project/doc/auto_tutorials/misc/plot_seeg.rst:61122: WARNING: undefined label: tut-viz-stcs:volume source estimates (if the link has no caption the label must precede a section header)
```

File: tutorials/misc/plot_seeg.py [hunk 1]
```diff
@@ -145,11 +145,12 @@
 stc = abs(stc)  # just look at magnitude
 clim = dict(kind='value', lims=np.percentile(abs(evoked.data), [10, 50, 75]))
 
-# plot Nutmeg style
-stc.plot(vol_src, subject=subject, subjects_dir=subjects_dir, clim=clim)
-
 ###############################################################################
 # Plot 3D source (brain region) visualization:
+#
+# The time course shown here is the `stc.data[idx]` where `idx` is the index
```
Comment (larsoner) [inline tutorials/misc/plot_seeg.py:151]: double backticks on code

File: tutorials/misc/plot_seeg.py [hunk 2]
```diff
@@ -162,3 +163,18 @@
 # You can save a movie like the one on our documentation website with:
 # brain.save_movie(time_dilation=3, interpolation='linear', framerate=10,
 #                  time_viewer=True, filename='./mne-test-seeg.m4')
+
+###############################################################################
+# In this tutorial, we used a BEM surface for the `fsaverage` subject from
```
Comment (larsoner) [inline tutorials/misc/plot_seeg.py:168]: double backticks on `fsaverage`

**Comment by agramfort** [comment]
@adam2392 when you want feedback for changes on documentation please share rendered html pages on circleci artifact

**Comment by adam2392** [comment]
Ah yes sorry. I was waiting for CI to become fixed, but just took a look and it doesn't seem to be related to this PR.

https://23154-1301584-gh.circle-artifacts.com/0/dev/auto_tutorials/misc/plot_seeg.html#sphx-glr-auto-tutorials-misc-plot-seeg-py

**Review by larsoner — APPROVED** [review]
@drammock feel free to review and merge if you're happy

File: tutorials/misc/plot_seeg.py [hunk 3]
```diff
@@ -145,11 +145,12 @@
...
 
-# plot Nutmeg style
-stc.plot(vol_src, subject=subject, subjects_dir=subjects_dir, clim=clim)
-
 ###############################################################################
 # Plot 3D source (brain region) visualization:
+#
+# The time course shown here is the ``stc.data[idx]`` where ``idx`` is the
+# index of the source with the largest absolute value across any time point.
+# In this example, it is simply the source with the largest raw signal value.
```
Comment (drammock) [inline tutorials/misc/plot_seeg.py:153]:
```suggestion
# By default, `stc.plot_3d() <mne.VolSourceEstimate.plot_3d>` will show a time
# course of the source with the largest absolute value across any time point.
# In this example, it is simply the source with the largest raw signal value.
# Its location is marked on the brain by a small blue sphere.
```

**Review by drammock — APPROVED** [review]
just one wording suggestion.

**Comment by larsoner** [comment]
Thanks @adam2392

**Pull request merged**